content/blog/20200130-jepsen.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Latest Jepsen Results against etcd 3.4.3
+draft: false
+date: 2020-01-30 17:00:00 -0000
+authors:
+- name: Xiang Li
+---
+
+Jepsen tested and analyzed etcd 3.4.3, and had both good results and useful feedback to share with us.
+
+<!--more-->
+
+A key part of etcd's design is strong consistency guarantees across the distributed key-value store. Kubernetes, Rook, OpenStack, and countless other critical software projects rely on etcd, in part, because of the etcd project's focus on reliability and correctness.
+
+Over the years, the etcd team has put tremendous effort on building [testing](https://coreos.com/blog/testing-distributed-systems-in-go.html) and [chaos engineering frameworks](https://coreos.com/blog/new-functional-testing-in-etcd.html).  We feel that we've improved our consistency, and have asked Jepsen for validation.
+
+[Jepsen](https://jepsen.io/) is the leading company testing open source distributed systems to check if they fulfill their consistency guarantees. They first tested etcd in version 0.4.1, and we have been working with them ever since to improve our results. In this blog post, we share the overall positive results of the Jepsen analysis of etcd v3.4.3 as well as our plans to improve the issues found in the analysis.
```

Comment:
"We've been working with them ever since" isn't quite right, if "them" refers to Jepsen as a company--our collaboration started on 2019-10-27. You've been using the test suite to guide development though, right?

Reply:
right. this needs to be clarified. we have been utilizing the tool Jepsen provided rather than working with the company directly.

Reply:
changing to "we've been using their findings ever since"

Reply:
maybe "making use of their tooling ever since"?